CHANGELOG.md

```diff
@@ -20,6 +20,12 @@
 
 ## Unreleased
 
+### Breaking Changes
+
+- Changes project layout to match `kubebuilder` `v4`. Some import paths have changed
+  e.g. `apis` -> `api`.
```

Comment:
I believe we should explicitly call out which changes are indeed breaking so `apis` -> `api` and `controllers` -> `controllers` (even though the latter is not really a part of an API that e.g. contains the types that one would normally use in a cluster but rather our controllers' logic.